docs/get-started/sign-up.md

```diff
@@ -33,20 +31,17 @@ To activate your Sumo Logic Trial account:
 1. In the email, click **Activate Now**.
 1. Fill out the **Activate Your Account** form, then click **Activate**.
 
-Next, you'll be taken to a brief two-step data collector installation process, which will get you up and running with your data in a couple of minutes.
+Thanks for signing up! Next, you'll be taken to a brief two-step data collector installation process, which will get you up and running with your data in a couple of minutes.
 
 ### Set up Collector
 
-1. Select the platform (Linux, Windows, macOS, or AWS) from which you'd like to collect data.<br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="Platform selection showing Linux, Windows, macOS, or AWS" width="400" />
+1. Select the platform (Linux, Windows, macOS, or AWS) from which you'd like to collect data.<br/><img src={useBaseUrl('img/get-started/data-collection.png')} alt="Platform selection showing Linux, Windows, macOS, or AWS" style={{border: '1px solid gray'}} width="400" />
 1. Copy, paste, and run the provided command to your PowerShell or Terminal. This will install the OpenTelemetry collector and relevant dashboards.
```

Comment:
they can also skip this step by going to the hamburger menu in the top right and choosing to skip.